models/Student.js

```diff
@@ -11,8 +10,13 @@ const StudentSchema = new Schema({
     type: String,
     required: true
   },
+  guardian_Name:{
+    type: String,
+    required: true
+  },
   grade: {
     type: String,
+    enum: ['6th', '7th', '8th', '9th', '10th', '11th', '12th', 'College Freshman', 'College Sophmore', 'College Junior', 'College Senior', 'Out of School'],
```

Comment:
We should store this list somewhere so we don't have to keep updating it in multiple places. We should be able to store it as a constant in the back-end, and then send it up to the front-end to populate the dropdowns.